JeffShepherd/Rancid-Tomatillos
# Final cleanup

## What does this PR do?

- Final cleanup of unused files, code comments, and whitespace

## How should it be tested in the UI?

- Run 'npm start' and do a full regression test of the application to ensure nothing with inadvertently changed.
